src/CoreBundle/Command/UpdateCountCommand.php

```diff
@@ -0,0 +1,51 @@
+<?php
+
+/*
+ * This file is part of the Chameleon System (https://www.chameleonsystem.com).
+ *
+ * (c) ESONO AG (https://www.esono.de)
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace ChameleonSystem\CoreBundle\Command;
+
+use Symfony\Component\Console\Command\Command;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Output\OutputInterface;
+
+/**
+ * Console command for counting updates that have not been executed yet.
+ */
+class UpdateCountCommand extends Command
```

Comment:
Could we rename the command to CountUpdateCommand to be in line with the existing commands (although your scheme would be better if we had used it from the beginning; in the future I'd like to move these commands to the DatabaseMigrationBundle where the current scheme would be OK)?

Reply:
ok